pkg/api/domains/domains_test.go

```diff
@@ -233,36 +233,79 @@ func createDomain(domainName string) v8weblogic.Domain {
 			Labels:    make(map[string]string),
 		},
 		Spec: v8weblogic.DomainSpec{
-			AdminServer:                         v8weblogic.AdminServer{},
+			AdminServer: v8weblogic.AdminServer{
+				AdminService: v8weblogic.AdminService{
+					Channels: func() []v8weblogic.Channel {
+						var channels []v8weblogic.Channel
+
+						channels = append(channels, v8weblogic.Channel{
+							ChannelName: "default",
+						})
+
+						channels = append(channels, v8weblogic.Channel{
+							ChannelName: "T3Channel",
+						})
+						return channels
+
+					}(),
+				},
+			},
 			AllowReplicasBelowMinDynClusterSize: false,
-			Clusters:                            nil,
-			ConfigOverrides:                     "",
-			ConfigOverrideSecrets:               nil,
-			Configuration:                       v8weblogic.Configuration{},
-			DataHome:                            "",
-			DomainHome:                          "",
-			DomainHomeInImage:                   false,
-			DomainHomeSourceType:                "",
-			DomainUID:                           "",
-			HttpAccessLogInLogHome:              false,
-			Image:                               "",
-			ImagePullPolicy:                     "",
-			ImagePullSecrets:                    nil,
-			IncludeServerOutInPodLog:            false,
-			IntrospectVersion:                   "",
-			LogHome:                             "",
-			LogHomeEnabled:                      false,
-			ManagedServers:                      nil,
-			MaxClusterConcurrentStartup:         0,
-			Replicas:                            nil,
-			RestartVersion:                      "",
-			ServerPod:                           v8weblogic.ServerPod{},
-			ServerService:                       v8weblogic.ServerService{},
-			ServerStartPolicy:                   "",
-			ServerStartState:                    "",
-			WebLogicCredentialsSecret:           corev1.SecretReference{},
+			Clusters: func() []v8weblogic.Cluster {
```

Comment:
looks like Clusters and AdminServer are the only things in v8oweblogic.DomainSpec that have actually changed. These changes don't look related to your foundByID ordering change...are they really needed?

Reply:
I am taking the opportunity to improve the coverage for the tests.  The coverage is only at 65% and the changes to the structures yield an additional 20%

Reply:
okay, but do you know why that is increasing the coverage? It's not quite clear to me